connection.php

```diff
@@ -0,0 +1,725 @@
+		$home_mobile = isset($input['input_data']['home_mobile']) ? $input['input_data']['home_mobile'] : ' ';
+		$home_landline = isset($input['input_data']['home_landline']) ? $input['input_data']['home_landline'] : ' ';
+		$home_fax = isset($input['input_data']['home_fax']) ? $input['input_data']['home_fax'] : ' ';
+		$office_street = isset($input['input_data']['office_street']) ? $input['input_data']['office_street'] : ' ';
+		$office_city = isset($input['input_data']['office_city']) ? $input['input_data']['office_city']: ' ';
+		$office_state = isset($input['input_data']['office_state']) ? $input['input_data']['office_state'] : ' ';
+		$office_zip = isset($input['input_data']['office_zip']) ? $input['input_data']['office_zip'] : ' ';
+		$office_mobile = isset($input['input_data']['office_mob']) ? $input['input_data']['office_mob'] : ' ';
+		$office_landline = isset($input['input_data']['office_landline']) ? $input['input_data']['office_landline'] : ' ';
+		$office_fax = isset($input['input_data']['office_fax']) ? $input['input_data']['office_fax'] : ' ';
+		$communication = implode(',' , $input['input_data']['communication']);
+	 	
+	 	//update operation on employee table
+	 	$update_employee = " UPDATE employee e
+		 	SET first_name = '$first_name',
+		 		middle_name = '$middle_name',
+		 		last_name = '$last_name',
+		 		prefix = '$prefix',
+		 		gender = '$gender',
+		 		dob = '$dob',
+		 		email_id = '$email_id',
+		 		password = '$password',
+		 		marital_status = '$marital_status',
+		 		employment = '$employment',
+		 		employer = '$employer',
+		 		image = '$image'
+		 	WHERE e.id = $employee_id";
+		 	
+		 $result_employee = mysqli_query($this->connect , $update_employee);
+	
+		 if($result_employee === TRUE)
+		 {
+		 	$update_residence = "UPDATE employee e
+		 		LEFT JOIN address addr ON e.id = addr.employee_id
+		 		SET street = '$home_street',
+		 			city = '$home_city',
+		 			state = '$home_state',
+		 			mobile = '$home_mobile',
+		 			landline = '$home_landline',
+		 			fax = '$home_fax'
+		 		WHERE addr.employee_id = $employee_id AND type = 'residence'";
+
+		 	$result_residence = mysqli_query($this->connect, $update_residence);
+		 		
+		 	$update_office = "UPDATE employee e
+		 		LEFT JOIN address addr ON e.id = addr.employee_id
+		 		SET street = '$office_street',
+		 			city = '$office_city',
+		 			state = '$office_state',
+		 			mobile = '$office_mobile',
+		 			landline = '$office_landline',
+		 			fax = '$office_fax'
+		 		WHERE addr.employee_id = $employee_id AND type = 'office'";
+
+		 	$result_office = mysqli_query($this->connect, $update_office);
+
+		 	$update_communication = "UPDATE employee e 
+		 		LEFT JOIN communication comm ON e.id = comm.employee_id
+		 		SET type = '$communication'
+		 		WHERE comm.employee_id = $employee_id";
+
+		 	$result_communication = mysqli_query($this->connect, $update_communication);
+		 }
+		 mysqli_close($this->connect);
```

Comment:
Always use a line break before and after code blocks/conditional blocks/looping blocks etc for better visibility of the codes and logics.